run/types/sessionIt.ts

```diff
@@ -56,10 +58,11 @@ export function iosIt(
 function mobileIt(
   platform: SupportedPlatformsType,
   title: string,
+  testRisk: testRisk,
   testToRun: (platform: SupportedPlatformsType) => Promise<void>,
   shouldSkip: boolean = false
 ) {
-  const testName = `${title} ${platform}`;
+  const testName = `${title} ${platform} @${testRisk}-risk`;
```

Comment:
why not use playwright annotations? https://playwright.dev/docs/test-annotations#tag-tests

Reply:
I thought the @-token in the test title acts as a test tag?

Reply:
yep all good, I didn't know playwright was scanning the testName for any @ and using it as tags